src/core_plugins/kibana/server/tutorials/apm/saved_objects/get_saved_objects.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import indexPattern from './index_pattern.json';
+import staticSavedObjects from './saved_objects.json';
+
+function getIndexPattern(server) {
+  let apmIndexPattern = 'apm*';
```

Comment:
We hard code this value in three places now.  Maybe time to refactor?

Also... it seems like we should never get here if the APM plugin isn't running?  The only way to stop it from running is to set it to disabled in kibana.yml, right?  In that case I can't even see these instructions.  In which case - why swallow the error if that flow isn't expected?

I don't think this could happen because I think disabling the plugin requires a restart of kibana, but if there was another way to get this to fail, it seems like it could be an error if there was a custom index name specified, but here the config service was temporarily down, so it falls back to `apm*` when it shouldn't.

If we *should* keep it in, is there any way to verify that the error is the one you want to skip? Otherwise we are potentially swallowing other errors and this might lead to hard to debug issues.

Reply:
The APM manifest gets loaded regardless of whether the APM app is running or not. I will try to add a check for the "not found error" to allow other errors to surface.